docs-src/content/cli/Transcribe.md

```diff
@@ -0,0 +1,160 @@
+---
+title: "Transcribe Command"
+weight: 20
+---
+
+This command is used for transcribing audio files.
+There are two modes: single file or list file.
+
+Usage:
+```
+transcribe FILE_PATH [flags]
+```
+
+In single mode, FILE_PATH should point to a single audio file.
+
+In list-file mode, FILE_PATH should point to a file listing multiple audio files with the format "Utterance_ID \t FILE_PATH \n". 
+    
+Audio files in the following formats are supported:
+    WAV, FLAC, MP3, VOX, and RAW(PCM16SLE).
+
+The file extension (wav, flac, mp3, vox, raw) will be used to determine which
+ codec to use.  Use WAV or FLAC for best results.
+
+<!--more-->
+
+| Short | Long form      | Arg | Purpose |
+| ----- | -------------- | --- | ------- |
+| -l | `--list-file` | none | Indicates that FILE_PATH is pointing to a list of audio files. |
+| -o | `--output` | string | Path to directory where the results should be written.<br/>"-" indicates stdout. (default "-") |
+| -m | `--model` | string | Selects which model ID to use for transcribing. (default "1") |
+| -c | `--audioChannels` | ints   | Audio channels to transcribe.  (Defaults to mono) |
+| | `--stereo` | none |  Sets `--audioChannels "0,1"` to transcribe both audio channels of a stereo file.|
+| -n | `--workers` | int | Number of concurrent requests to send to cubicsvr. (default 1) |
+| -f | `--outputFormat` | string | Format of output.  Can be [json,json-pretty,timeline,utterance-json]. (default "timeline") |
+| -a | `--fmt.timeline.maxAlts` | int | Maximum number of alternatives to provide for each result in the timeline format. (default 1) |
+
+
+### List-file
+
+To process multiple files, use the `--list-file` flag and specify a text file instead of an audio file.
+The text file should be tab-delimited, with one audio file per line.  Each line should begin with a unique
+identifier containing no white space, which will be used to identify which transcript belongs to which 
+input file.  
```

Comment:
Do we need to mention what happens if they use the same name more than once?  Or mention that the behavior is undefined?